tests/ProxyManagerTest/Functional/LazyLoadingGhostFunctionalTest.php

```diff
@@ -829,4 +830,37 @@ public function getPropertyAccessProxies()
             ],
         ];
     }
+
+    /**
+     * @dataProvider skipPropertiesFixture
+     *
+     * @param string   $className
+     * @param string   $propertyName
+     * @param string[] $properties
+     */
+    public function testSkipProperties($className, $propertyName, $properties)
+    {
+        $factory      = new LazyLoadingGhostFactory();
```

Comment:
See the other tests in here as an example. Then turn the factory test into a unit-test (via mocking)